demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -222,3 +226,72 @@ def test_IsAskConditionHasUnhandledReplyOptionsValidator():
         )
     }
     assert IsAskConditionHasUnhandledReplyOptionsValidator().is_valid([playbook])
+
+
+def test_indicator_pb_must_stop_on_error():
+    """
+    Given: A pb with queryEntity indicators
+    When: Playbook stops on error
+    Then: Validation should pass
+    """
+    playbook = create_playbook_object(
+        ["inputs"],
+        [
+            [
+                {
+                    "value": {},
+                    "required": False,
+                    "description": "",
+                    "playbookInputQuery": {"query": "", "queryEntity": "indicators"},
+                }
+            ],
+        ],
+    )
+    res = IsStoppingOnErrorValidator().is_valid([playbook])
+    assert len(res) == 0
+    playbook.tasks = {
+        "0": TaskConfig(
+            **{
+                "id": "test task",
+                "taskid": "27b9c747-b883-4878-8b60-7f352098a631",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"no": ["1"]},
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+            }
+        )
+    }
+    res = IsStoppingOnErrorValidator().is_valid([playbook])
+    assert len(res) == 0
+
+
+def test_indicator_pb_must_stop_on_error_invalid():
+    playbook = create_playbook_object(
+        ["inputs"],
+        [
+            [
+                {
+                    "value": {},
+                    "required": False,
+                    "description": "",
+                    "playbookInputQuery": {"query": "", "queryEntity": "indicators"},
+                }
+            ],
+        ],
+    )
+    playbook.tasks = {
+        "0": TaskConfig(
+            **{
+                "id": "test task",
+                "taskid": "27b9c747-b883-4878-8b60-7f352098a631",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"no": ["1"]},
+                "continueonerror": True,
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+
+            }
+        )
+    }
```

Comment:
Why not adding this to create_playbook_object.

Reply:
Done